autogpt/commands/image_gen.py

```diff
@@ -76,12 +81,13 @@ def generate_image_with_hf(prompt: str, filename: str) -> str:
     return f"Saved to disk:{filename}"
 
 
-def generate_image_with_dalle(prompt: str, filename: str) -> str:
+def generate_image_with_dalle(prompt: str, filename: str, image_size: str) -> str:
     """Generate an image with DALL-E.
 
     Args:
         prompt (str): The prompt to use
         filename (str): The filename to save the image to
+        image_size (str): The image_size
```

Comment:
I think the parameter is missing size: int, please see the existing code is referring to size. Will be fixed in #2491.
```
    if size not in [256, 512, 1024]:
closest = min([256, 512, 1024], key=lambda x: abs(x - size))
```

Reply:
Thanks @kinance I'll close the PR !